Dockerfile-dev

```diff
@@ -7,20 +7,15 @@ ENV OPENSLIDES_BACKEND_DEBUG 1
 
 WORKDIR /srv/code
 
-RUN apk add --no-cache gcc musl-dev linux-headers git \
```

Comment:
Note: For the dev setup, we do not need an extra user, since changes in this container (with correct mounted volumes) should reflect on the host system, e.g. when running black